src/flex-example-app-android.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<application xmlns="http://ns.adobe.com/air/application/19.0">
+    <id>com.example.android.snake</id>
```

Comment:
we'll want to use a different default bundle ID and have the README suggest changing this. similar to the app ID